doc/opf/powermodels.rst

```diff
@@ -6,33 +6,50 @@ Optimization with PowerModels.jl
 
+4. Test your `PowerModels <https://lanl-ansi.github.io/PowerModels.jl/stable/#Installation-1>`_ installation by executing :code:`test PowerModels`. Then, test if calling Python from Julia works as described `here <https://github.com/JuliaPy/PyCall.jl#usage>`_.
 
-#. Configure Python to be able to call Julia 
+.. note:: If you cannot plot using matplotlib and Anaconda in Julia, add the environment variable :code:`QT_PLUGIN_PATH` with value e.g. :code:`"C:\Anaconda3\Library\plugins"` to your system path.
+
+5. To call Julia from Python, install the pyjulia package with :code:`pip install julia`. Afterwards, test if everything works by importing PowerModels from Python with: :code:`from julia.PowerModels import run_ac_opf`. This takes some time, since Python starts a julia instance in the background, but it if the import completes without error everything is configured correctly and you can now use PowerModels to optimize pandapower networks.
+
+.. warning:: **Windows 10 only:** A bug currently makes it necessary to modify a file of the pyjulia package. The exact line which needs to be modified depends on how you installed pyjulia. To find the exact line, have a look at your traceback after you execute the command :code:`from julia import Main`, e.g. in Pycharm (Corresponding `Github Issue <https://github.com/JuliaPy/pyjulia/issues/67#issuecomment-508407747>`_):
+
+    .. code-block::
+
+        File "C:\python\Anaconda3\lib\site-packages\julia\core.py", line 686, in __init__
+        self.libjulia = ctypes.PyDLL(libjulia_path, ctypes.RTLD_GLOBAL)
+        File "C:\python\Anaconda3\lib\ctypes\__init__.py", line 356, in __init__
+        self._handle = _dlopen(self._name, mode)
+        OSError: [WinError 127] The specified procedure could not be found
+
+    In this case, it is line no. 686. Replace
+
+    .. code-block::
+
+        self.libjulia = ctypes.PyDLL(libjulia_path, ctypes.RTLD_GLOBAL)
+
+    with
+
+    .. code-block::
+
+        cwd = os.getcwd()
+        os.chdir(os.path.dirname(libjulia_path))
+        self.libjulia = ctypes.PyDLL(libjulia_path, ctypes.RTLD_GLOBAL)
+        os.chdir(cwd)
+
+    Now Julia should at least work in Spyder, but not in Pycharm.
 
-    - Add the Julia binary folder (e.g. /Julia-1.1.0/bin) to the `system variable PATH <https://www.computerhope.com/issues/ch000549.htm>`_
-    - Install pyjulia with :code:`pip install julia`
-    - test if everything works by importing PowerModels from Python with: :code:`from julia.PowerModels import run_ac_opf`. This takes some time, since Python starts a julia instance in the background, but it if the import completes without error everything is configured correctly and you can now use PowerModels to optimize pandapower networks.
-      .. note::
-        Calling julia inside of python only works on Windows machines. Under Linux you can use "python-jl yoursript.py" in a terminal.

```

Comment:
This line might not be needed in the documentation since a detailed error message appears on linux:

`
Your Python interpreter "/opt/miniconda3/bin/python3"
is statically linked to libpython.  Currently, PyJulia does not fully
support such Python interpreter.
The easiest workaround is to pass `compiled_modules=False` to `Julia`
constructor.  To do so, first *reboot* your Python REPL (if this happened
inside an interactive session) and then evaluate:
    >>> from julia.api import Julia
    >>> jl = Julia(compiled_modules=False)
Another workaround is to run your Python script with `python-jl`
command bundled in PyJulia.  You can simply do:
    $ python-jl PATH/TO/YOUR/SCRIPT.py
See `python-jl --help` for more information.
For more information, see:
    https://pyjulia.readthedocs.io/en/latest/troubleshooting.html

`

Reply:
So do you still want to change that before merging?

Reply:
I think line 34+35 are not needed (current version), and I removed them in my commit. My last comment was a clarification on why i think those lines are not needed. From my perspective, it's good to go.